mender/templates/secret-s3-artifacts.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     helm.sh/chart: "{{ .Chart.Name }}"
 data:
   AWS_URI: {{ .Values.global.s3.AWS_URI | default .Values.global.url | default (ternary (printf "https://%s" .Values.api_gateway.service.name ) (printf "http://%s" .Values.api_gateway.service.name) (.Values.api_gateway.env.SSL)) | b64enc }}
+  AWS_EXTERNAL_URI: {{ .Values.global.s3.AWS_EXTERNAL_URI | default .Values.global.s3.AWS_URI | default .Values.global.url | default (ternary (printf "https://%s" .Values.api_gateway.service.name ) (printf "http://%s" .Values.api_gateway.service.name) (.Values.api_gateway.env.SSL)) | b64enc }}
```

Comment:
The external value is falcultative.
Maybe it's easier to put nothing if we don't want to use it?

```yaml
AWS_EXTERNAL_URI: {{ .Values.global.s3.AWS_EXTERNAL_URI | default "" }}
```

or:

```yaml
AWS_EXTERNAL_URI: {{ .Values.global.s3.AWS_EXTERNAL_URI }}
```

Or:

```yaml
{{- if .Values.global.s3.AWS_EXTERNAL_URI }}
AWS_EXTERNAL_URI: {{ .Values.global.s3.AWS_EXTERNAL_URI }}
{{- else }}
```